execute/plugin.go

```diff
@@ -2,42 +2,59 @@ package execute
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"slices"
 	"sort"
 	"sync/atomic"
 	"time"
 
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
 	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
 	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
 
-	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+	"github.com/smartcontractkit/chainlink-ccip/internal/libs/slicelib"
 )
 
+// maxReportSizeBytes that should be returned as an execution report payload.
+const maxReportSizeBytes = 250_000
```

Comment:
I think this is too big if this report will be transmitted onchain, this is 250 kilobytes which is >> the max size of a non-empty ethereum transaction of 46kB (although this is unlikely, for it to be all nonzero bytes).

Reply:
This value was selected because it's the same one v1 uses. Should I change it to 46_000?